CONTRIBUTORS.md

```diff
@@ -6,7 +6,7 @@ Contributors
 <!-- Please add your badges to .all-contibutorsrc -->
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-266-orange.svg)](#contributors)
+[![All Contributors](https://img.shields.io/badge/all_contributors-267-orange.svg)](#contributors)
```

Comment:
Looks like this change got incorporated that I wasn't aware. @fkiraly should I resubmit this PR?

Reply:
no, that's automatic, it's fine